### DISCUSSION TITLE
JS code containing evaluation of ```location``` is not deployed.

### DISCUSSION THREAD

**Discussion by SebastianKapunkt** [community/community#40269] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm writing a small game and want the user to be able to adjust configuration. That configuration is represented as URL parameters. To read that parameters, I want to utilize the ```window.location``` [API](https://developer.mozilla.org/en-US/docs/Web/API/Location).

When I write a small JavaScript script that contains this code (or any code that evaluates the location object):

```window.location.search```

It is never deployed/updated. When I change HTML content in the same commit, the HTML is actually updated.

There is no error message in the deployment or anywhere else. The GitHub Action just says, everything went fine.

Is this a hidden security feature?

@ossanna16 thanks a lot for your quick answer! I wanted to follow up on this because I feel there is undocumented behavior in GitHub pages. So basically I wrote valid JavaScript code that is silently not deployed by the GitHub action because - as it seems to me - I accessed the ```window.location``` object.

**Comment by ossanna16** [comment]
Hi @SebastianKapunkt,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on JavaScript. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
